assets/onboarding/query-string-router/url-functions.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Add query string to URL with pushState.
+ *
+ * @param {string} queryStringName  Query string name to be added to the URL.
+ * @param {string} queryStringValue Query string value to be added to the URL.
+ */
+const pushQueryStringState = ( queryStringName, queryStringValue ) => {
+	const { search } = window.location;
+	const searchParams = new URLSearchParams( search );
+
+	searchParams.set( queryStringName, queryStringValue );
+	window.history.pushState( {}, '', `?${ searchParams.toString() }` );
+};
+
+/**
+ * Get query string from URL.
+ *
+ * @param {string} name  Name of the query string to get.
+ *
+ * @return {string} The value in the query string.
+ */
+const getQueryString = ( name ) =>
```

Comment:
Just a naming thing, query string is usually the whole `?x=a&y=b` part of the URL, maybe this should be called query string field or **query param**? Or something that's short enough but meaningful :)

Reply:
Makes sense! Updated here: https://github.com/Automattic/sensei/pull/3085/commits/4df842b186e89ebbf0baf9989582496ebc6c8df7

I updated the function to `getParam` and some variables to `paramName` and `paramValue`.